### ISSUE TITLE
Improve saved questions organization

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#3497]
This is an über issue based on feedback from Raphael:

Questions metadata
 - [ ] I should know if a question is used by a Pulse or Dashboard (so no one edits by mistake)
 - [ ]  A question's label(s) should show up on the Question page as well
 - [ ] Rafael suggested being able to "publish a query (vs keeping it in WIP mode)", but we're discussing a similar thought different framework where certain privileged users would be able to mark certain saved questions (and other artifacts) as "Official" or "Verified."

Search/filtering
 - [ ] Be able to filter questions by table they're based on
 - [ ] Questions' descriptions should be visible in the saved questions list
 - [ ] Be able to search by "created by [user name]" / "modified by [user name]" 
 - [ ] Ability to reorder labels to change the order in which they are displayed

**Comment by moumny** [comment]
Slightly related, automatically add labels by table. (https://github.com/metabase/metabase/issues/3375)

In general its super hard to search trough questions. Weve observed from users that the questions tab is not very used. Instead users find question in dashboard and modify them from there.

**Comment by mazameli** [comment]
Can you give us any additional context, moumny? Were there other things that make the questions list hard for your users to search through that aren't listed in this issue?

**Comment by moumny** [comment]
Filtering by table should be enough. My idea was to add a label based on the table. In our case nobody adds labels its just too time consuming.

The other issue with searching for question is that the more descriptive the name is, the harder is it to fit in the dashboard card. So my suggestion was to allow editing of card's title so we can keep questions name meaningful. Its now possible since 0.20 to add a title in most charts, but not in multiseries (the name of question is in the legend this time)

The last issue with the question listing page is that usually it ends up as 20x the same questions but with different filters and granularity (hour day week etc..) or charting settings. Maybe we're using it wrong ?

**Comment by tlrobinson** [comment]
@mazameli Didn't we have a separate "topics" section that was based on tables in the original design? That sounds similar to the table labels idea.

**Comment by kdoh** [comment]
@tlrobinson Yeah in the original version there was a notion of trying to infer topics based on the table names "users", etc.

**Comment by tlrobinson** [comment]
@moumny Regarding the problem of creating multiple similar cards with different filters and granularities, would #3575 help? We'd also need to allow you to parameterize the granularity.

**Comment by mazameli** [comment]
Another helpful thing would be a way to filter by questions without a label (so that you can then label them).

**Comment by brunobergher** [comment]
More of a repository for a discussion and an old one at that than an issue (or list of issues), closing,
